<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fm_estoque_editing_page</class>
 <widget class="QDialog" name="Fm_estoque_editing_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbl_prodCod">
         <property name="text">
          <string>Cod. Produto</string>
         </property>
         <property name="buddy">
          <cstring>le_prodCod</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_prodCod"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_prodName">
         <property name="text">
          <string>Nome do Produto</string>
         </property>
         <property name="buddy">
          <cstring>le_prodName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_prodName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lbl_prodDesc">
         <property name="text">
          <string>Descrição do Produto</string>
         </property>
         <property name="buddy">
          <cstring>le_prodDesc</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_prodDesc"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="lbl_prodQtt">
           <property name="text">
            <string>Quantidade em Estoque</string>
           </property>
           <property name="buddy">
            <cstring>le_prodQtt</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_prodQtt"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="lbl_bValue">
           <property name="text">
            <string>Valor de Compra</string>
           </property>
           <property name="buddy">
            <cstring>dsb_bValue</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_bValue"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="lbl_sValue">
           <property name="text">
            <string>Valor de Venda</string>
           </property>
           <property name="buddy">
            <cstring>dsb_sValue</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_sValue"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="lbl_supplier">
           <property name="text">
            <string>Fornecedor</string>
           </property>
           <property name="buddy">
            <cstring>cb_supplier</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_supplier"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btt_att">
         <property name="text">
          <string>Atualizar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btt_cancelar">
         <property name="text">
          <string>Cancelar</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
